components/grid/grouping/aggregates.md

```diff
@@ -120,13 +124,17 @@ To enable aggregates:
             <GroupFooterTemplate>
                 @*access the aggregates of the ActiveProjects column*@
                 All active projects: @context.Sum
-                <br />
-                @*access the aggregates of the other columns*@
-                Total teams: @context.AggregateResults[nameof(Employee.Team)].Count
-                <br />
-                Total employees: @context.AggregateResults[nameof(Employee.Name)].Count
-                <br />
-                Average salary: @context.AggregateResults[nameof(Employee.Salary)].Average.Value.ToString("C0")             
+
+                @* access the aggregates of the other columns if any *@
+                @if (context.AggregateResults.Any())
```

Comment:
Not sure what triggered this change, but if we want this code to be bullet-proof, we should check each aggregate key separately.